docs/ocis/development/testing.md

```diff
@@ -539,6 +539,57 @@ BEHAT_FEATURE="tests/acceptance/features/apiAntivirus/antivirus.feature" \
 make test-acceptance-api
 ```
 
+## Running Test Suite With Federated Sharing (@ocm)
+Test suites that are tagged with `@ocm` require running two different ocis instances. More detailed information and configuration related to it can be found [here](https://doc.owncloud.com/ocis/5.0/deployment/services/s-list/ocm.html).
+
+Put
+```bash
+127.0.0.1 ocis-server
+127.0.0.1 federation-ocis-server
+```
+in the `/etc/hosts` file
+
+### Setup first ocis instance
+
+```bash
+# run oCIS
+OCIS_URL="https://ocis-server:9200" \
+PROXY_ENABLE_BASIC_AUTH=true \
+GRAPH_INCLUDE_OCM_SHAREES=true \
+OCM_OCM_INVITE_MANAGER_INSECURE=true \
+OCM_OCM_SHARE_PROVIDER_INSECURE=true \
+OCM_OCM_STORAGE_PROVIDER_INSECURE=true \
+OCM_OCM_PROVIDER_AUTHORIZER_PROVIDERS_FILE="${workspaceFolder}/tests/config/drone/providers.json" \
+OCIS_ADD_RUN_SERVICES="ocm"
+ocis/bin/ocis server
+```
+
+The federation-ocis-server will be available at: https://ocis-server:9200/
+
+### Setup second ocis instance
+#### Using .vscode/launch.json
+
+The federation-ocis-server will be available at: https://federation-ocis-server:10200/
```

Comment:
Do we have other alternatives for running a second OCIS instance? Maybe some information on that could be useful.  
I can see more detail on running the first ocis instance but limited information on the setup of the second ocis instance. Maybe more info could be nice

Reply:
I can write more detail how to run ocis using launch.json https://github.com/owncloud/ocis/pull/9439/files#diff-bd5430ee7c51dc892a67b3f2829d1f5b6d223f0fd48b82322cfd45baf9f5e945

Here in docs I cannot put all needed envs (long list). I tried to create .env file and run ocis as source `.env && ocis/bin/ocis server ` but it doesn't work for me https://github.com/owncloud/ocis/issues/9461

Reply:
found my mistake. I'll add `.env` file and description how to run  `federation-ocis-server`